### PR TITLE
[AC-2997] fix: change root element of feedback modal

### PR DESCRIPTION
**Pull request by bang9**
## Changes
update root element id of message feedback modal

ticket: [AC-2997]

[AC-2997]: https://sendbird.atlassian.net/browse/AC-2997?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/BotMessageFeedback.tsx [hunk 1]
```diff
@@ -116,6 +117,7 @@ function BotMessageFeedback({ message }: { message: CoreMessageType }) {
           message?.myFeedback?.rating &&
           showFeedbackModal && (
             <MessageFeedbackModal
+              rootElementId={elementIds.widgetWindow}
```
Comment (chohongm) [inline src/components/BotMessageFeedback.tsx]: does this property exsit in MessageFeedbackModalProps?

Comment (chohongm) [reply src/components/BotMessageFeedback.tsx]: @bang9 `sendbird-modal__content` is being overwritten by widget.  It sets
  `max-width: calc(100% - 80px);` which is required for `MessageFeedbackFailedModal` but not for `MessageFeedbackModal`. 
  
  
Screenshot 2024-07-01 at 5 59 48 PM

To workaround this,

we could add `MessageFeedbackModal.contentClassName` and change `Modal.contentClassName` to `Modal.contentClassNames`.

Comment (bang9) [reply src/components/BotMessageFeedback.tsx]: > does this property exsit in MessageFeedbackModalProps?

No, I've added this to [uikit](https://github.com/sendbird/sendbird-uikit-react/compare/37951504a629bf4efd1f2903dcb5d42dc0a800a6...8f26cda305b87dd2b8a3fcfc558488d42d2bd485#diff-7b6184c31c6596319f7a88c49300984b3c42b635b03648df23dbd2af19aded88)

Comment (bang9) [reply src/components/BotMessageFeedback.tsx]: > @bang9 `sendbird-modal__content` is being overwritten by widget. It sets `max-width: calc(100% - 80px);` which is required for `MessageFeedbackFailedModal` but not for `MessageFeedbackModal`.

@liamcho It seems that the design of all modals has 40px padding on the left and right sides.
image

Comment (chohongm) [reply src/components/BotMessageFeedback.tsx]: the original implementation did not have 40px. it was smaller so the buttons were not overlaping like above. I [asked designer for answer](https://sendbird.slack.com/archives/C0772Q0BB8Q/p1719833180010909)

**Comment by bang9** [comment]
@liamcho The UI was already in place, but there were some issues with implementing the business logic. I have resolved those issues

https://github.com/sendbird/chat-ai-widget/assets/26326015/50aef82d-0edb-4987-ab36-940a1f83e46c


https://github.com/sendbird/chat-ai-widget/assets/26326015/cba3489c-f4f4-42e9-ab4e-f45536a76fc3

**Review by chohongm — CHANGES_REQUESTED** [review]
@bang9 아래 수정 부탁드립니다
Screenshot 2024-07-02 at 4 00 05 PM
need to be inside widget
Screenshot 2024-07-02 at 4 00 52 PM
need to have smaller side margins so buttons are not overlapping each other

**Review by chohongm — APPROVED** [review]
LGTM

**Pull request merged**